rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -106,6 +109,7 @@ public abstract class SystemProperties {
     public static final String PROPERTY_PERSIST_BLOOMS_CACHE_SNAPSHOT = "cache.blooms.persist-snapshot";
 
     /* Testing */
+    public static final String LAST_KNEW_PEERS_FILE = "lastKnewPeers.txt";
```

Comment:
what do you think about using some well-known / established data format, for instance, java `.properties` with `<nodeId>: <address>` mapping?

Reply:
As it was for internal usage only I didn't think about any specific format.

Reply:
yeah, we already use `.properties` in the db folder for other needs, so I just thought that it would be a good idea to continue using it. `.txt` seemed to me more like a textual document rather than a storage file